main.nf

```diff
@@ -385,39 +580,15 @@ process multiqc {
     rtitle = custom_runName ? "--title \"$custom_runName\"" : ''
     rfilename = custom_runName ? "--filename " + custom_runName.replaceAll('\\W','_').replaceAll('_+','_') + "_multiqc_report" : ''
     """
-    multiqc -f $rtitle $rfilename --config $multiqc_config .
+    multiqc -f ${rtitle} ${rfilename} --config ${multiqc_config} .
     """
 }
-<<<<<<< HEAD
-//
-//
-// /*
-//  * STEP 3 - Output Description HTML
-//  */
-// process output_documentation {
-//     tag "$prefix"
-//     publishDir "${params.outdir}/Documentation", mode: 'copy'
-//
-//     input:
-//     file output_docs
-//
-//     output:
-//     file "results_description.html"
-//
-//     script:
-//     """
-//     markdown_to_html.r $output_docs results_description.html
-//     """
-// }
-=======
-
 
 
 /*
- * STEP 3 - Output Description HTML
+ * STEP 5 - Output Description HTML
  */
 process output_documentation {
-    tag "$prefix"
     publishDir "${params.outdir}/Documentation", mode: 'copy'
 
     input:
```

Comment:
This looks a bit broken here... 

Also, once you put in the template into a channel, input needs to use that channel :-)

Reply:
Sam as for the channel and output info